src/openiddict-test/project.json

```diff
@@ -32,7 +31,9 @@
     "Microsoft.Extensions.Configuration.Json": "1.0.0",
     "Microsoft.Extensions.Logging.Console": "1.0.0",
     "Microsoft.Extensions.Logging.Debug": "1.0.0",
-    "Microsoft.NETCore.Platforms": "1.0.1"
+    "Microsoft.NETCore.Platforms": "1.0.1",
+    "AspNet.Security.OAuth.Validation": 1.0.0-alpha2-final,
```

Comment:
Bah! Where's the intellisense!😭😭
On Jul 7, 2016 4:10 PM, "Kévin Chalet" <notifications@github.com> wrote:

> In src/openiddict-test/project.json
> <https://github.com/capesean/openiddict-test/pull/19#discussion_r69996939>
> :
>
> > @@ -32,7 +31,9 @@
> >      "Microsoft.Extensions.Configuration.Json": "1.0.0",
> >      "Microsoft.Extensions.Logging.Console": "1.0.0",
> >      "Microsoft.Extensions.Logging.Debug": "1.0.0",
> > -    "Microsoft.NETCore.Platforms": "1.0.1"
> > +    "Microsoft.NETCore.Platforms": "1.0.1",
> > +    "AspNet.Security.OAuth.Validation": 1.0.0-alpha2-final,
>
> Invalid JSON spotted 😄
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/capesean/openiddict-test/pull/19/files/0832ae147ab6237c8f13807002037aa560c38d63#r69996939>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIMjLG00zVEyGfQqps_apETZMr_C58prks5qTXlMgaJpZM4JFp_g>
> .
>